charts/otel-collector/templates/ingress.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.ingress.enabled -}}
+  {{- $ingressApiIsStable := eq (include "ingress.isStable" .) "true" -}}
+  {{- $ingressSupportsIngressClassName := eq (include "ingress.supportsIngressClassName" .) "true" -}}
+  {{- $ingressSupportsPathType := eq (include "ingress.supportsPathType" .) "true" -}}
+  {{- $serviceName := include "otel-collector.fullname" . }}
+  {{- $servicePort := .Values.service.collector.port -}}
+  {{- $ingressPath := .Values.ingress.path -}}
+  {{- $ingressPathType := .Values.ingress.pathType -}}
+  {{- $extraPaths := .Values.ingress.extraPaths -}}
+apiVersion: {{ template "ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  {{- if .Values.ingress.annotations }}
+annotations:
+  {{ toYaml .Values.ingress.annotations | indent 4 }}
+  {{- end }}
+labels:
+  {{- include "otel-collector.labels" . | nindent 4 }}
+  {{- range $key, $value := .Values.ingress.extraLabels }}
+  {{ $key }}: {{ $value }}
+  {{- end }}
+name: {{ template "otel-collector.fullname" . }}
+namespace: {{ .Release.Namespace | quote }}
+spec:
+  {{- if and $ingressSupportsIngressClassName .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName }}
+  {{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+  {{- $url := splitList "/" . }}
+  - host: {{ first $url }}
+    http:
+      paths:
+  {{ if $extraPaths }}
+  {{ toYaml $extraPaths | indent 10 }}
+  {{- end }}
+- path: {{ $ingressPath }}
+  {{- if $ingressSupportsPathType }}
+  pathType: {{ $ingressPathType }}
+  {{- end }}
+  backend:
+    {{- if $ingressApiIsStable }}
+    service:
+      name: {{ $serviceName }}
+      port:
+        number: {{ $servicePort }}
+    {{- else }}
+    serviceName: {{ $serviceName }}
+    servicePort: {{ $servicePort }}
```

Comment:
@arramos84 otel collector may need to expose 2 ports one for sending the traces and another may be for prom to scrape the metrics.
How do I add 2 ports here?